awslimits/settings.py

```diff
@@ -7,16 +8,18 @@
 WTF_CSRF_SECRET_KET = SECRET_KEY
 
 ROLE_ARN = os.environ.get("ROLE_ARN")
+ACCOUNT_ID = re.findall('\d+', ROLE_ARN)[0]
+ACCOUNT_ROLE = ROLE_ARN.split('/')[-1]
 REGION_NAME = os.environ.get("REGION_NAME", "us-east-1")
+PREMIUM_ACCOUNT = int(os.environ.get("PREMIUM_ACCOUNT", 0))
 
 assert ROLE_ARN, "Need to pass a role ARN"
 
 LIMIT_ALERT_PERCENTAGE = int(os.environ.get("LIMIT_ALERT_PERCENTAGE", 90))
 
 SENDGRID_API_KEY = os.environ.get("SENDGRID_API_KEY")
 EMAIL_RECIPIENTS = os.environ.get('EMAIL_RECIPIENTS')
-FROM_EMAIL_ADDRESS = os.environ.get('FROM_EMAIL_ADDRESS')
+FROM_EMAIL_ADDRESS = os.environ.get('FROM_EMAIL_ADDRESS', 'awslimits@alerts.com')
```

Comment:
It's a dummy email, which is what the email address comes in as. It doesn't have to be real and doesn't matter what it is. Without one set the `sg.client.mail.send.post(request_body=data)` fails